### PR TITLE
Multiple recipient emails

### PR DESCRIPTION
**Pull request by alexkb0009**
Update Extension.php to allow multiple emails in YAML config as recipients. 
Addresses issue - "Address in mailbox ... does not comply with RFC 2822, 3.6.2." for multiple recipients.

### CODE REVIEW

**Comment by jadwigo** [comment]
I'm very reluctant to merge this
- I think this makes the configuration of forms more complex
- the input fields on the form suddenly have a comma separated hidden feature that will also be exposed to the people filling in the form which might lead to abuse
- also it's missing the documentation for the new option

If multiple email addresses should be possible I would want a better structure in the `.yml` than a simple comma separated string - which would mean refactoring it a bit to use name, emailaddress pairs instead of the two fields that are used now, something like the following

```
recipients: [ { name: "Test User", email: test@example.com }, { name: "Another User", email: another@example.com }]
```

This would need some more work before it's ready to merge

**Comment by alexkb0009** [comment]
Thx for feedback!
Check out new edits re: your response. 
Btw how the heck do I pull/merge your master into mine re: updates to master (in GitHub)? Don't want to git clone locally as my working directory is already in a different (Bolt instance) repository.

**Comment by jadwigo** [comment]
by the way: https://help.github.com/articles/syncing-a-fork/ and alternative options: http://stackoverflow.com/questions/7244321/how-to-update-github-forked-repository

**Comment by jadwigo** [comment]
@alexkb0009 can you redo the changes on a clean fork? this is a bit messy now and hard to follow.

**Comment by alexkb0009** [comment]
Re-branched + pull-requested @ https://github.com/jadwigo/SimpleForms/pull/9
